jaeger_client/codecs.py

```diff
@@ -110,15 +113,27 @@ def extract(self, carrier):
                 if self.url_encoding:
                     value = urllib_parse.unquote(value)
                 debug_id = value
-        if not trace_id and baggage:
-            raise SpanContextCorruptedException('baggage without trace ctx')
```

Comment:
I could not find any rationalization for why this exception had to be thrown in this scenario.